chanzuckerberg/single-cell-data-portal
# feat(wmg): Add instrumentation to Source Data button on Gene Expressi…

…on #4000

## Reason for Change

- #4000

## Changes

- add
    - Add analytics when source data is opened and when "our documentation" link is clicked
- remove
- modify

## Testing steps

1. Navigate to gene expression
2. Click source data
3. Ensure that analytic is being sent in console
4. Close sidebar
5. Select dataset(s)
6. Click source data
7. Ensure that analytic is sent for selected dataset(s)
8. Click "our documentation" link under Methodology
9. Ensure analytic is sent when link is clicked

## Notes for Reviewer
